lib/registration.pm

```diff
@@ -567,6 +567,9 @@ sub fill_in_registration_data {
         if (get_var('SCC_URL') || get_var('SMT_URL')) {
             push @tags, 'untrusted-ca-cert';
         }
+        # The SLE15-SP2 license page moved after registration.
+        push @tags, 'license-agreement'          if (!get_var('MEDIA_UPGRADE') && is_sle('15-SP2+'));
```

Comment:
should add some handle for new tag in while loop?

Reply:
Thanks. Missing the code here. Updated.